app/src/components/SubTitle/SubTitle.spec.tsx

```diff
@@ -0,0 +1,34 @@
+import React from "react";
+import { shallow } from "enzyme";
+
+import SubTitle from "./SubTitle";
+
+describe("SubTitle", () => {
+  describe("className tests", () => {
+    const customClass = "custom-class";
+    it("should render SubTitle without custom class", () => {
+      const wrapper = shallow(<SubTitle />);
+      expect(wrapper.find("h2").hasClass(customClass)).toBe(false);
+    });
+    it("should render SubTitle with custom class", () => {
+      const wrapper = shallow(<SubTitle className={customClass} />);
+      expect(wrapper.find("h2").hasClass(customClass)).toBe(true);
+    });
+  });
+
+  describe("children tests", () => {
+    const customText = "Custom text";
+    it("should render SubTitle with custom text", () => {
+      const wrapper = shallow(<SubTitle>{customText}</SubTitle>);
+      expect(wrapper.find("h2").text()).toMatch(customText);
+    });
+    it("should render SubTitle with custom child paragraph", () => {
+      const wrapper = shallow(
+        <SubTitle>
+          <p>{customText}</p>
+        </SubTitle>,
+      );
+      expect(wrapper.find("h2").find("p").text()).toMatch(customText);
```

Comment:
this tests just the text content of tag, which was tested above, how about checking the overall html content? `children().html()` https://enzymejs.github.io/enzyme/docs/api/ShallowWrapper/html.html

Reply:
good idea, on it 🦅